lib/tasks/user.rake

```diff
@@ -9,9 +9,14 @@ namespace :user do
   desc 'Convert old transactions to balance transactions'
   task convert_transactions: :environment do
     Transaction.find_each do |transaction|
-      balance_transaction = BalanceTransaction.create(comment: nil)
-      transaction.update(transactinable_type: BalanceTransaction,
-                         transactinable_id: balance_transaction.id)
+      balance_transaction = BalanceTransaction.create(
+        comment: transaction.comment,
+        date: transaction.date
+      )
+      transaction.update(
+        transactinable_type: BalanceTransaction,
+        transactinable_id: balance_transaction.id
+      )
```

Comment:
@sinpai what do you think about using `BalanceTransactions::Creator` here?

Reply:
@AleksSenkou I think it will be insufficient. In this case we are trying to add Balance transaction to the existing transaction. But with use of `BalanceTransactions::Creator` another transaction would be created. `BalanceTransactions::Updater` needs existing BalanceTransaction to operate. Using services here to perform this task I need to change services slightly. Should I try to use services here or it can be left without changes?